src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -1966,9 +1966,13 @@ private String getResourceFile(final String uri, final String contentType) throw
         return result;
     }
 
-    private void storePluginPropertiesAsParams(final Map<String, String> pluginProperties, final Multimap<String, String> params) {
+    protected void storePluginPropertiesAsParams(final Map<String, String> pluginProperties, final Multimap<String, String> params) {
         for (final String key : pluginProperties.keySet()) {
             params.put(JaxrsResource.QUERY_PLUGIN_PROPERTY, String.format("%s=%s", UTF8UrlEncoder.encode(key), UTF8UrlEncoder.encode(pluginProperties.get(key))));
         }
     }
+
+    protected KillBillHttpClient getKillBillHttpClient(){
```

Comment:
Moving this would be good, so as not to mix PRs.

We should probably also think it through, i.e. decide which methods should be `protected` (once we do it, they become part of a public API, so we have less flexibility to change them). I'd also be curious to understand the use-case (i.e. why you need it).